### PR TITLE
Sort functionality

### PR DESCRIPTION
**Pull request by shrimeichock**
- Allow user to sort by price & alphabetical
- Allow user to filter by genre, publisher and author

TO DO:
- Allow user to filter by price ranges
- Add tests for controller

### CODE REVIEW

File: src/main/java/bookstore/inventory/SortCriteria.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * Criteria for sorting books by
+ * @author Shrimei Chock
+ */
+
+public enum SortCriteria {
+    LOW_TO_HIGH("low_to_high"),
+    HIGH_TO_LOW("high_to_low"),
+    ALPHABETICAL("alphabetical");
+
```
Comment (thanujasiva) [inline src/main/java/bookstore/inventory/SortCriteria.java:12]: Could we also have a NONE option, to clear any of the existing sort values?

**Review by thanujasiva — APPROVED** [review]

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -51,10 +61,47 @@ <h1>Online Book Store</h1>
...
+            </span>
+
+            <h3>Publisher</h3>
+            <span th:each="publisher : ${publishers}" >
+                <label th:for="${'checkbox-' + publisher}" th:text="${publisher}"></label>
+                <input type="checkbox" th:id="${'checkbox-' + publisher}" th:name="publisher" th:value="${publisher}" />
+                <br>
+            </span>
+
+        </div>
```
Comment (sabahsamwatin) [inline src/main/resources/templates/home.html:104]: The layout format can be improved a little.

**Review by sabahsamwatin — APPROVED** [review]

File: src/main/resources/templates/home.html (hunk 2 above)
Comment (shrimeichock) [reply src/main/resources/templates/home.html:104]: next milestone

File: src/main/java/bookstore/inventory/SortCriteria.java (hunk 1 above)
Comment (shrimeichock) [reply src/main/java/bookstore/inventory/SortCriteria.java:12]: next milestone https://github.com/users/waheebh1/projects/1/views/2?pane=issue&itemId=45720732

**Comment by shrimeichock** [comment]
https://github.com/waheebh1/online-bookstore/issues/47

**Comment by shrimeichock** [comment]
https://github.com/waheebh1/online-bookstore/issues/49

**Pull request merged**